vertexai/snippets/src/test/java/vertexai/gemini/SnippetsIT.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2023 Google LLC
+ * Copyright 2024 Google LLC
```

Comment:
The copyright year refers to when the file was initially created. Hence, please revert the year.

Reply:
Good to know, I'll revert.